packages/auth/src/Auth.ts

```diff
@@ -94,6 +93,8 @@ export class AuthClass {
 	private _storageSync;
 	private oAuthFlowInProgress: boolean = false;
 
+	amplify = Amplify;
```

Comment:
We can't rely on `amplify` to be a service registry / container because it's not aware that `Credentials` is a required dependency of `Auth`. (i.e. Instantiating `Auth` needs to also create `Credentials`)

```suggestion
	credentials = Credentials;
```